_opensearch/java-rest-high-level.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
I suspect we should put this under Clients and tools rather than OpenSearch, but let me know what you think.